reference/display.md

```diff
@@ -16,6 +16,7 @@ It can model an embedded screen or it can display any graphical information such
 
 If the first child of the [Display](#display) node is or contains (recursive search if the first node is a [Group](group.md)) a [Shape](shape.md) node having a [ImageTexture](imagetexture.md), then the internal texture of the(se) [ImageTexture](imagetexture.md) node(s) is replaced by the texture of the [Display](#display).
```

Comment:
it seems that here there is a missing `or` between Group and Shape. But this is not related to the merge.